terrain/perlin.cc

```diff
@@ -0,0 +1,64 @@
+#include <stdlib.h>
+#include <stdio.h>
+#include <math.h>
+
+namespace Perlin {
```

Comment:
Can you link where you got this in a comment?

Reply:
https://rosettacode.org/wiki/Perlin_noise

Reply:
Yes, but in the code. Not here.